index.d.ts

```diff
@@ -54,6 +54,8 @@ export interface HandleResponsesOptions {
 	alwaysServeDocs?: boolean
 
 	specOutputFileBehavior: SPEC_OUTPUT_FILE_BEHAVIOR | string
+
+  swaggerDocumentOptions?: object
```

Comment:
an `object` type is discouraged. We should either infer it from the swagger project, or create our own types based on [1]. The first scenario is better and I'll see if they have the types to make it work.

[1] https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md

Reply:
thanks @kiprasmel  for feedback,
i read swagger-ui configuration and found `Array<string>` is the better type to be defined